### ISSUE TITLE
Consider always "resizing" JPG and PNGs to strip private exif, geo, etc. data

### ISSUE THREAD

**Issue by humphd** [mozilla/thimble.mozilla.org#2344]
We added automatic image resizing code for images >250K.  One of the nice side effects of this, is that we also now automatically strip any EXIF data from JPGs, or text/data chunks from PNG.  This is important so users don't accidentally overshare location information when dragging an image into a project.

My question is whether we should do this same thing for images under 250K, and instead of resizing, just do a single 1:1 pass, but also strip out any extra data that users are unknowingly sharing.

cc @Pomax

**Comment by Pomax** [comment]
That's not a bad plan, but an unintended side effect is that stripping EXIF also removes copyright and license information. In some countries, that means the image is now "free for all", and in others it means it's now "all rights reserved"... do we want to try to sort that out as well?

**Comment by humphd** [comment]
I think the benefits of stripping fingerprinting data from images outweighs any non-web usable, embedded copyright info that an image might have, given our target audience.

**Comment by Pomax** [comment]
I disagree with the "non-web usable" part (what we're actually seeing is a harsh lack in proper copyright acknowledgement by browsers, which *should* be exposing this information with a simple context menu option when used in conjunction with media resources) but I can get behind the "this is irrelevant for our audience" part.

The resize should be fast for images that are already under 250kb, so I don't expect this to be a noticable perf. hit except on really slow systems, which I suspect will not be able to run Thimble itself in a responsive manner anyway.

**Comment by humphd** [comment]
I think we'll have to alter our code slightly for the <250K case.  I tried it and it made small images larger, which isn't our goal for sure.
